fixchain/fix_and_log_test.go

```diff
@@ -305,11 +322,17 @@ func testFixAndLogQueueChain(t *testing.T, i int, test *fixAndLogTest, fl *FixAn
 }
 
 func TestFixAndLogQueueChain(t *testing.T) {
+	ctx := context.Background()
 	for i, test := range fixAndLogQueueTests {
 		f := &Fixer{toFix: make(chan *toFix)}
+		c := http.Client{Transport: &testRoundTripper{}}
```

Comment:
And here, and in other tests files in this PR (so I don't keep writing 'and here') :)

Reply:
(I use 'passim' for this sentiment)